manifests/overlays/standalone/kustomization.yaml

```diff
@@ -3,10 +3,8 @@ kind: Kustomization
 namespace: kubeflow
 resources:
   - ../../base
-  - ../../manager
-  - ../../rbac
   - namespace.yaml
 images:
-  - name: kubeflow/training-operator:v1.0.0
-    newName: deepakmuley/kubeflow-training-operator
-    newTag: "1"
+  - name: kubeflow/training-operator
+    newName: kubeflow/training-operator
+    newTag: "46c586463e3c13b0d4f2d534aae84a46329dc4ef"
```

Comment:
@Jeffwan just for my knowledge, could you please document where did we get this image newTag from.

Reply:
found it here. https://hub.docker.com/r/kubeflow/training-operator/tags?page=1&ordering=last_updated

Reply:
Sure. This is a temporary solution. Once post-submit job is fixed, we will change back to AWS public ECR because docker hub free account will be throttled.  If ECR doesn't work for us, I will document this step